src/spicy/spicyz/driver.cc

```diff
@@ -195,6 +213,8 @@ hilti::Result<hilti::Nothing> Driver::compile() {
 
     SPICY_DEBUG("Running Spicy driver");
 
+    _error = hilti::Nothing();
```

Comment:
Is it correct that any previous `_error` value would only matter if `compile` is invoked multiple times which shouldn't happen? In that case I'd simply return an error here instead of silently resetting it, e.g.,
```suggestion
    if ( _error )
        return hilti::result::Error(hilti::util::fmt("compilation triggered with previous error: %s", _error.error()));
```